en/flight_modes_fw/stabilized.md

```diff
@@ -1,32 +1,49 @@
+It is much easier to fly than [Manual mode](#manual-mode) because you can't roll or flip it, and if needed it is easy to level the vehicle (by centering the control sticks).
 :::
 
-The vehicle will glide if the throttle is lowered to 0% (motor stops). In order to perform a turn the command must beheld throughout the maneuver because if the roll is released the plane will stop turning and level itself (the same is true for pitch and yaw commands).
+The vehicle climb/descends based on pitch input and performs a [coordinated turn](https://en.wikipedia.org/wiki/Coordinated_flight) if the roll/pitch sticks are non-zero.
+Roll and pitch are angle controlled (you can't roll upside down or loop).
+
+The vehicle will glide if the throttle is lowered to 0% (motor stops).
+In order to perform a turn the command must be held throughout the maneuver because if the roll is released the plane will stop turning and level itself (the same is true for pitch and yaw commands).
 
 The diagram below shows the mode behaviour visually (for a [mode 2 transmitter](../getting_started/rc_transmitter_receiver.md#transmitter_modes)).
 
-![FW Manual Flight](../../assets/flight_modes/manual_stabilized_FW.png)
+![FW Manual Flight](../../assets/flight_modes/stabilized_fw.png)
 
 ## Technical Description
 
-RC/manual mode where centered RP sticks level vehicle.
+Manual mode where centered roll/pitch sticks levels vehicle attitude.
+The vehicle course and altitude are not maintained, and can drift due to wind.
 
-- Centered sticks put vehicle into straight and level flight. The vehicle course and altitude are not maintained, and can drift due to wind.
+- Centered Roll/Pitch/Yaw sticks (inside deadband) put vehicle into straight and level flight.
+  The vehicle course and altitude are not maintained, and can drift due to wind.
 - If roll/pitch sticks are non-zero the vehicle does a coordinated turn (manual yaw input is added to rudder control input to control sideslip).
+- Pitch stick controls pitch angle.
+- Roll stick controls roll angle.
+  Autopilot will maintain <a href="https://en.wikipedia.org/wiki/Coordinated_flight">coordinated flight</a>.
+- Throttle stick controls throttle.
+- Yaw stick adds an additional yaw rate setpoint (signal will be added to the one calculated by the autopilot to maintain [coordinated flight](https://en.wikipedia.org/wiki/Coordinated_flight)).
+- Manual control input is required (such as RC control, joystick).
+  - Roll: Assistance from autopilot to stabilize the attitude.
+    Position of stick maps to the orientation of vehicle.
+  - Pitch: Assistance from autopilot to stabilize the attitude.
+    Position of stick maps to the orientation of vehicle.
+  - Yaw: Stick input is sent directly to control allocation.
+  - Throttle: Stick input is sent directly to control allocation.
 
 ## Parameters
 
 | Parameter | Description |
 | --------- | ----------- |
```

Comment:
@sfuhrer What params affect position mode? Is it the same ones as documented for altitude - https://docs.px4.io/main/en/flight_modes_fw/altitude.html#parameters

Reply:
addressed in https://github.com/PX4/PX4-user_guide/pull/3046